app.R

```diff
@@ -1,47 +1,97 @@
 #
-# This is a Shiny web application. You can run the application by clicking
-# the 'Run App' button above.
+# Raw data from SCBI_ForestGEO dataset
+# specifically, tree census data (2008, 2013, 2018) and tree ecology data
 #
-# Find out more about building applications with Shiny here:
-#
-#    http://shiny.rstudio.com/
+# censusJoin is a dataframe defined in MP3_eking03.Rmd
 #
 
+
 library(shiny)
+library(ggplot2)
```

Comment:
you're going to need to load all the necessary packages here for this app to work, not just ggplot2